htdocs/comm/index.php

```diff
@@ -168,10 +169,7 @@
 		$total = 0;
 		$num = $db->num_rows($resql);
 
-		print '<div class="div-table-responsive-no-min">';
-		print '<table class="noborder centpercent">';
-		print '<tr class="liste_titre">';
-		print '<th colspan="3">'.$langs->trans("ProposalsDraft").' <a href="'.DOL_URL_ROOT.'/comm/propal/list.php?search_status=0"><span class="badge">'.$num.'</span></a></th></tr>';
+		StartSimpleTableHeader("ProposalsDraft", "comm/propal/list.php", "search_status=0", 2, $num);
```

Comment:
If you introduce a external function to open a div and table, you must also introduce a function on same level that close the div and the table.
Having closing tag into a php page must have the opening on same level. If opening is into a sublevel, we must also close in a sublevel.